### ISSUE TITLE
TypeError: lineRef.current.getTotalLength is not a function

### ISSUE THREAD

**Issue by RomanPanaget** [Eliav2/react-xarrows#72]
[comment]: <> (replace paragraphs text where needed)
**Describe the bug & Expected behavior**
TypeError: lineRef.current.getTotalLength is not a function
--> getTotalLength is no longer supported

**To Reproduce**
Launch create-react-app, install react-xarrows and use it somewhere. Run yarn upgrade if the error doesn't show up.

**Comment by Eliav2** [comment]
Well I need to add `as any` at the end of [this] (https://github.com/Eliav2/react-xarrows/blob/2dd35ac1439d210ca741b3c66bd0437c490e2436/src/index.tsx#L755) line. I will do so

**Comment by Eliav2** [comment]
fixed with v1.7.1

**Comment by KutnerUri** [comment]
getting this error during tests - 
```
TypeError: _c.getTotalLength is not a function

...
      at Object.getPosition (node_modules/.pnpm/registry.npmjs.org+react-xarrows@2.0.2_react@16.13.1/node_modules/react-xarrows/lib/index.js:1124:90)
      at node_modules/.pnpm/registry.npmjs.org+react-xarrows@2.0.2_react@16.13.1/node_modules/react-xarrows/lib/index.js:180:37
      at commitHookEffectListMount (node_modules/.pnpm/registry.npmjs.org+react-dom@16.13.1_react@16.13.1/node_modules/react-dom/cjs/react-dom.development.js:19731:26)
```

version: 2.0.2
using Jest and react testing library.
